### PR TITLE
Lambda functions removed from examples

### LINKED ISSUE #1512: Fix examples on Windows
We sometimes use lambdas in the examples when creating torch datasets: https://github.com/lightly-ai/lightly/blob/e7f13a53e68031ee0ae77124e76401b48381001f/examples/pytorch/mae.py#L87

This results in issues on Windows as lambdas cannot be pickled. We should update the examples and replace the lambdas with pickleable functions.

### PR DESCRIPTION
**Pull request by snehilchatterjee**
**Enhancement:**
- Replaced lambda functions with named functions in the examples.

**Fixes Issue:**
- Closes #1512.

**Approach:**
- I created a bash script to list all files containing lambda functions. After identifying the files, I manually replaced the lambda functions with named functions.

### CODE REVIEW

**Comment by snehilchatterjee** [comment]
Hey @guarin any ideas what is causing the above error ?

File: examples/pytorch/aim.py [hunk 1]
```diff
@@ -65,11 +65,17 @@ def forward(self, images):
 
 transform = AIMTransform()
 # we ignore object detection annotations by setting target_transform to return 0
+
+
+def target_transform_function(t):
```
Comment (guarin) [inline examples/pytorch/aim.py:70]: Can we call this just `target_transform`? It keeps the examples a bit more concise and matches the `target_transform` argument name from the dataset.

**Review by guarin — COMMENTED** [review]
Thanks a lot for the PR!

File: examples/pytorch/aim.py (hunk 1 above)
Comment (snehilchatterjee) [reply examples/pytorch/aim.py:70]: yeah sure I will update it right now

**Review by guarin — COMMENTED** [review]
This looks great! Thanks so much.

**Review by guarin — APPROVED** [review]

**Comment by snehilchatterjee** [comment]
> This looks great! Thanks so much.

Glad I could contribute :smile:

**Pull request merged**